srcpkgs/ibus-libthai/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'ibus-libthai'
+pkgname=ibus-libthai
+version=0.1.4
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="ibus-devel libthai gtk+3-devel"
+depends="ibus libthai gtk+3"
```

Comment:
This line is superfluous, because xbps-src detects these dependencies automatically.